client/src/components/LoginTest/register.js

```diff
@@ -1,7 +1,6 @@
 // import React from "react";
 import React, {
-    Component,
-    Fragment
+       Fragment
```

Comment:
same here, i'm not sure if anyone else needs this. Please confirm @Spacejnk @czjordan323